### PR TITLE
Feat: add closed captions support

### PR DESCRIPTION
**Pull request by kslimani**
It adds closed captions support introduced in Clappr release 0.2.68.

To test, just run dev server with `npm start` command, the angel-one video has subtitles.

### CODE REVIEW

File: src/clappr-dash-shaka-playback.js [hunk 1]
```diff
@@ -123,6 +123,88 @@ class DashShakaPlayback extends HTML5Video {
...
+    }
+
+    let tracks = this.closedCaptionsTracks
+    let showingTrack
+
+    // Note: -1 is for hide all tracks
+    if (trackId !== -1) {
+      showingTrack = tracks.find(track => track.id === trackId)
+      if (!showingTrack) {
+        return // Track id not found
```
Comment (leandromoreira) [inline src/clappr-dash-shaka-playback.js]: Maybe we could `log` this message.

File: src/clappr-dash-shaka-playback.js [hunk 2]
```diff
@@ -123,6 +123,88 @@ class DashShakaPlayback extends HTML5Video {
...
+      }
+      if (this._shakaTTVisible && showingTrack.track.active === true) {
+        return // Track already showing
```
Comment (leandromoreira) [inline src/clappr-dash-shaka-playback.js]: Dido, but for sure, not showing up all of them on the console, maybe using INFO severity.

File: public/index.html [hunk 3]
```diff
@@ -44,7 +44,7 @@
             })
         }
     </script>
-    <script src="https://cdn.jsdelivr.net/clappr/latest/clappr.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/gh/clappr/clappr@latest/dist/clappr.min.js"></script>
```
Comment (leandromoreira) [inline public/index.html]: Cool, can you also update the README.md to instruct users to use this URL?

File: src/clappr-dash-shaka-playback.js (hunk 1 above)
Comment (kslimani) [inline src/clappr-dash-shaka-playback.js]: which severity for this one ? "warning" ?

**Comment by kslimani** [comment]
I updated readme and added track Log messages. I used "warn" level for track not found and "info" level for track already showing.

**Comment by kslimani** [comment]
Also updated CDN JSDELIVR URL for plugin.

**Comment by leandromoreira** [comment]
Thank you very much @kslimani

**Pull request merged**